tools/compare_v_performance_between_commits

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e 
+
+if (( $# != 2 )); then
+    echo "Usage: compare_v_to_c_performance COMMIT_BEFORE COMMIT_AFTER"
+    exit
+fi
+
+######################################################################
+## NB: This tool depends that you have the following cli instruments:
+##
+##     bash
+##     git
+##     cp
+##     stat
+##     strip
+##     upx
+##     make
+##     hyperfine
+##     cc -> working, recent, sane C99 compiler (both gcc/clang work)
+##
```

Comment:
```sh
type git >/dev/null 2>&1 || { printf 'ERR missing "git"\n' >&2; exit 1; }
type upx >/dev/null 2>&1 || { printf 'ERR missing "upx"\n' >&2; exit 1; }
type hyperfine >/dev/null 2>&1 || { printf 'ERR missing "hyperfine"\n' >&2; exit 1; }
```
(`cp` `make` `strip` are POSIX; instead of `cc` POSIX defines `c99` which'll be mentioned below; instead of non-POSIX `stat` one can use `wc -c` as mentioned below)

Reply:
cc is for the Makefile/BSDmakefile that v uses to bootstrap itself. I should have mentioned that in the comment, thanks.

Reply:
Using type to check for missing dependencies is very neat. Thanks.